src/components/layout/nav/signup/SignUp.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useRef } from "react";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Form from "react-bootstrap/Form";
+import "./SignUp.scss";
+import { NavLink } from "react-router-dom";
+
+const SignUp = () => {
+  const passwordOfUser = useRef(false);
+  const emailOfUser = useRef(false);
+
+  // TODO(emrerdem1): This function will send the input values to check auth with 'emailOfUser.current.value' and 'passwordOfUser.current.value';
+  const handleLoginInputs = () => null;
+
+  return (
+    <>
+      <Col xl={11} className="signupBox signUpBoxResponsive">
```

Comment:
do there need to be to classes, `x and xResponsive`? it seems to me that they would simply be the same rule, except that media queries change what happens